cucumber/authenticators_oidc/features/authn_oidc_okta.feature

```diff
@@ -15,21 +19,26 @@ Feature: OIDC Authenticator V2 - Users can authenticate with Okta using OIDC
           - !variable client-id
           - !variable client-secret
           - !variable claim-mapping
-          - !variable state
```

Comment:
Do we need to think about policy migration for customers that may have values for state and nonce already defined in policy or will these just be ignored?  Should we have a test for this?